README.md

```diff
@@ -101,7 +101,7 @@ fn main() {
     let spec = serde_json::from_reader(file).unwrap();
     let mut generator = progenitor::Generator::default();
 
-    let content = generator.generate_text(&spec).unwrap();
+    let content = generator.generate_tokens(&spec).unwrap().to_string();
 
```

Comment:
Can you add in the formatting code we now recommend with syn and prettyprint? You can copy it from the example build.rs

Reply:
Excellent. Thank you!